addon/components/imput-money.js

```diff
@@ -42,6 +42,10 @@ export default TextField.extend({
     this._super(...arguments);
   },
 
+  refresh: observer('options', function() {
+    this.$().maskMoney('mask');
```

Comment:
This line could be put after the line 51?

Reply:
Yes, you're right. Sorry I didn't look close enough at the existing code.